development/docker-compose-tests.yml

```
@@ -31,6 +62,22 @@ x-php: &service-php
+      <<: *service-php-8-4-build
+      tags:
+        - srt-reader-php-8-4:latest
+    entrypoint: ["make", "php-syntax-busybox"]
+
+  php-8-5:
+    <<: *service-php
+    build:
+      <<: *service-php-build
+      tags:
+        - srt-reader-php-8-5:latest
+    entrypoint: [ "make", "php-syntax-busybox" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)